app/main/forms/frameworks.py

```diff
@@ -0,0 +1,31 @@
+from flask.ext.wtf import Form
+from wtforms import validators, StringField
+from wtforms import BooleanField as _BooleanField
+
+
+class BooleanField(_BooleanField):
+    """Tri-state boolean field
```

Comment:
That's what we have in the [frontend toolkit](http://alphagov.github.io/digitalmarketplace-frontend-toolkit/forms/selection-buttons.html#offlineWorking-yes). That field can be either yes, no or neither.